tests/Utils/SimpleDiffTest.php

```diff
@@ -16,7 +16,7 @@
 use PHPUnit\Framework\TestCase;
 use SimpleThings\EntityAudit\Utils\SimpleDiff;
 
-class SimpleDiffTest extends TestCase
+final class SimpleDiffTest extends TestCase
```

Comment:
Could you please add the return type declaration to `dataDiff()` method and the argument type declarations to `testDiff()`?

Reply:
Done.